docs/content/concepts/ops-jobs-graphs/metadata-tags.mdx

```diff
@@ -0,0 +1,109 @@
+
+```python file=/concepts/ops_jobs_graphs/metadata_tags.py startafter=start_metadata_on_job endbefore=end_metadata_on_job
+@op
+def my_op():
+    return "Hello World!"
+
+@job(
+    metadata={
+        "owner": "data team", # will be converted to MetadataValue.text
+        "docs": MetadataValue.url("https://docs.dagster.io")
+    }
+)
+def my_job_with_metadata():
+    my_op()
+```
+
+In addition to adding metadata on the `@job` decorator, you can also add metadata using the <PyObject object="GraphDefinition" method="to_job" /> method.
+
+```python file=/concepts/ops_jobs_graphs/metadata_tags.py startafter=start_metadata_on_graph_to_job endbefore=end_metadata_on_graph_to_job
+@graph
+def my_graph():
+    my_op()
+
+my_second_job_with_metadata = my_graph.to_job(metadata={"owner": "api team", "docs": MetadataValue.url("https://docs.dagster.io")})
+```
+
+### Viewing Metadata
+
+After attaching metadata to a job, you can view it in dagit by navigating to the job overview page. Metadata will be displayed in the right pane.
+
+<img
+alt="job-metadata.png"
+src="/images/concepts/ops-jobs-graphs/job-metadata.png"
+/>
+
+## Tags
+
+Tags allow you to attach information to the run created when you execute a job. Tags can contain any information you want, and dagster will also attach some tags to your runs (we'll cover these later).
+
+### Specifying Tags
+
+You can specify tags you want attached to every run by adding them to a job. Tags are specified as a dictionary of key value pairs where the key must be a string and the value must be a string or json that is serializable to a string.
+
+```python file=/concepts/ops_jobs_graphs/metadata_tags.py startafter=start_tags_on_job endbefore=end_tags_on_job
+@job(
+    tags={
+        "foo": "bar"
+    }
+)
+def my_job_with_tags():
+    my_op()
+```
+
+In addition to adding tags on the `@job` decorator, you can also add metadata using the <PyObject object="GraphDefinition" method="to_job" /> method.
+
+```python file=/concepts/ops_jobs_graphs/metadata_tags.py startafter=start_tags_on_graph_to_job endbefore=end_tags_on_graph_to_job
+my_second_job_with_tags = my_graph.to_job(tags={"foo": "bar"})
+```
+
+When executing a job, you can add tags to the run using the Launchpad in Dagit <img
+alt="tag-adder.png"
+src="/images/concepts/ops-jobs-graphs/tag-adder.png"
+/>
+
+### Viewing Tags
+
+You can view the tags that have been attached to runs by going to the Runs page in Dagit <img
+alt="tags-viewer.png"
+src="/images/concepts/ops-jobs-graphs/tags-viewer.png"
+/>
+
+### Dagster provided tags
+
+Dagster will automatically add tags to your runs in some cases including:
+
+- The solid selection for the run, if applicable
+- The partition set and partition of the run, if applicable
+- The schedule that triggered the run, if applicable
+- The parent run of a re-executed run
+- OTHER THINGS
```

Comment:
@sryza @dpeng817 any other instances when we add tags?

Reply:
backfill id, resume/retry, docker image tag